hawk/app/models/cib.rb

```diff
@@ -457,32 +504,32 @@ def initialize(id, user, use_file = false)
 
     # Special-case defaults for properties we always want to see
     @crm_config = Hashie::Mash.new(
-      :"cluster-infrastructure"       => _('Unknown'),
-      :"dc-version"                   => _('Unknown'),
-      :"stonith-enabled"              => true,
-      :"symmetric-cluster"            => true,
-      :"no-quorum-policy"             => 'stop'
+      cluster_infrastructure: _('Unknown'),
+      dc_version: _('Unknown'),
+      stonith_enabled: true,
+      symmetric_cluster: true,
+      no_quorum_policy: 'stop',
     )
 
     # Pull in everything else
     # TODO(should): This gloms together all cluster property sets; really
     # probably only want cib-bootstrap-options?
     @xml.elements.each('cib/configuration/crm_config//nvpair') do |p|
-      @crm_config[p.attributes['name'].to_sym] = CibTools.get_xml_attr(p, 'value')
+      @crm_config[p.attributes['name'].underscore.to_sym] = CibTools.get_xml_attr(p, 'value')
     end
 
     @rsc_defaults = Hashie::Mash.new
     @xml.elements.each('cib/configuration/rsc_defaults//nvpair') do |p|
-      @rsc_defaults[p.attributes['name'].to_sym] = CibTools.get_xml_attr(p, 'value')
+      @rsc_defaults[p.attributes['name'].underscore.to_sym] = CibTools.get_xml_attr(p, 'value')
     end
 
     @op_defaults = Hashie::Mash.new
     @xml.elements.each('cib/configuration/op_defaults//nvpair') do |p|
-      @op_defaults[p.attributes['name'].to_sym] = CibTools.get_xml_attr(p, 'value')
+      @op_defaults[p.attributes['name'].underscore.to_sym] = CibTools.get_xml_attr(p, 'value')
     end
 
     is_managed_default = true
-    if @crm_config.has_key?(:"is-managed-default") && !@crm_config[:"is-managed-default"]
+    if @crm_config.has_key?(:is_managed_default) && !@crm_config[:is_managed_default]
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`.